src/chia_log/parsers/block_parser.py

```diff
@@ -27,7 +27,7 @@ class BlockParser:
     def __init__(self):
         logging.info("Enabled parser for block found stats.")
         self._regex = re.compile(
-            r"([0-9:.]*) full_node (?:src|chia).full_node.full_node\s*: INFO\s* ((?:🍀|.)\s* Farmed unfinished_block)"
+            r"([0-9:.]*) full_node (?:src|chia).full_node.full_node\s*: INFO\s* ((?:🍀 ️|.)\s*Farmed unfinished_block)"
```

Comment:
What's this character after the 🍀? Looks like empty space but it's not. Otherwise `\s*` will cover it.

It'd be good if anyone else can confirm if they have that in their output before we merge this. Also adding an example log and test-case for that parser will prevent similar issues in future. Similar to [already existing log tests](https://github.com/martomi/chiadog/tree/main/tests/chia_log/logs)

Reply:
Seems to be this character https://www.fileformat.info/info/unicode/char/fe0f/index.htm